CI fails for slow tests: OSError: You are trying to access a gated repo

CI fails for slow tests: https://github.com/huggingface/trl/actions/runs/18522980131/job/52787020203
> OSError: You are trying to access a gated repo
Make sure to have access to it at https://huggingface.co/google/gemma-3n-E2B-it.
401 Client Error. (Request ID: Root=1-68ef62a0-63f3c4056cb46c2169cf2218;65732a89-ffad-4eff-acfb-2e69edcf215c)
Cannot access gated repo for url https://huggingface.co/google/gemma-3n-E2B-it/resolve/main/config.json.
Access to model google/gemma-3n-E2B-it is restricted. You must have access to it and be authenticated to access it. Please log in.

```python
FAILED tests/test_sft_trainer.py::TestSFTTrainer::test_train_vlm_gemma_3n - OSError: You are trying to access a gated repo.
Make sure to have access to it at https://huggingface.co/google/gemma-3n-E2B-it.
401 Client Error. (Request ID: Root=1-68ef62a0-63f3c4056cb46c2169cf2218;65732a89-ffad-4eff-acfb-2e69edcf215c)

Cannot access gated repo for url https://huggingface.co/google/gemma-3n-E2B-it/resolve/main/config.json.
Access to model google/gemma-3n-E2B-it is restricted. You must have access to it and be authenticated to access it. Please log in.
```

--- a/tests/test_sft_trainer.py
+++ b/tests/test_sft_trainer.py
@@ -1477,6 +1477,7 @@ class TestSFTTrainer(TrlTestCase):
     # To ensure coverage, we run tests on the full model but mark them as slow to exclude from default runs.
     @pytest.mark.slow
     @require_vision
+    @pytest.mark.skip(reason="Model google/gemma-3n-E2B-it is gated and requires HF token")
     def test_train_vlm_gemma_3n(self):
         # Get the dataset
         dataset = load_dataset("trl-internal-testing/zen-image", "conversational_language_modeling", split="train")

